samples/sensor/tofsense/README.rst

```diff
@@ -0,0 +1,45 @@
+.. zephyr:code-sample:: tofsense
+   :name: TOFSense Time Of Flight sensor
+   :relevant-api: sensor_interface
+
+   Get distance data from a TOFSense sensor (interrupt mode).
+
+Overview
+********
+
+This sample periodically measures distance between TOFSense sensor
+and target. The result is displayed on the console.
+
+Requirements
+************
+
+This sample uses the TOFSense sensor with its factory default settings, on the UART interface.
+
+References
+**********
+
+ - Datasheet: https://ftp.nooploop.com/downloads/tofsense/TOFSense_Datasheet_V3.0_en.pdf
+ - User manual: https://ftp.nooploop.com/downloads/tofsense/TOFSense_User_Manual_V3.0_en.pdf
```

Comment:
Sensor samples should be agnostic to the specific driver implementation. It's ok to mention specific examples, but the README as currently written assumes the nooploop is the only type of ToF sensor.